index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+   <main>
+        <nav>
+            <ul>
+                <li><a href="#"><div></div></a></li>
+                <span></span>
+                <li><a href="#">Store</a></li>
```

Comment:
It's better for the apple logo to be set as `<img>` tag [check here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image)